src/server/mvf4.cljs

```diff
@@ -0,0 +1,46 @@
+(ns server.mvf4)
+
+; TODO
+; Find the user and format the rap-sheet to
+; match this structure
+(defn get-user-rapsheet
+  [user-id]
+  {:warning 3 :kick 3 :temp-ban 3 :perm-ban 0})
+
+(def punishment-config {
+                        :warning  3
+                        :kick     3
+                        :temp-ban 3
+                        :perm-ban 1
+                        })
+
+(def punishment-levels [:warning :kick :temp-ban :perm-ban])
+
+(defn find-punishment
+  ([rap-sheet config]
+   (find-punishment 0 rap-sheet config))
+  ([level rap-sheet config]
+   (let [level-k (get punishment-levels level)]
+     (if (< (level-k rap-sheet) (level-k config))
+       level-k
+       (recur (inc level) rap-sheet config)))))
+
+; TODO
+; Write discord actions
+(def punishments
+  {
+   :warning  (fn [msg-obj] (.reply msg-obj "You have been warned"))
```

Comment:
I've implemented this in mvf 1 with: 
`(.reply msg " wash your mouth out with soap! We don't use that kind of language here.")`

Reply:
It'll fit in better here I guess